anxcloud/struct_vsphere_locations.go

```diff
@@ -0,0 +1,34 @@
+package anxcloud
```

Comment:
could you move that to `struct_locations.go` file to have each location in one file? That would help later with refactoring.

Reply:
done